webapp/frontend/lib/services/trade_constraints_service.g.dart

```diff
@@ -7,8 +7,8 @@ part of 'trade_constraints_service.dart';
 // **************************************************************************
 
 TradeConstraints _$TradeConstraintsFromJson(Map<String, dynamic> json) => TradeConstraints(
-      maxLocalMarginSats: json['max_local_margin_sats'] as int,
-      maxCounterpartyMarginSats: json['max_counterparty_margin_sats'] as int,
+      maxLocalBalanceSats: json['max_local_balance_sats'] as int,
```

Comment:
Did it work to generate these files or did you edit it manually?

Reply:
Edited manually!

Reply:
I've also run it with `fvm dart run build_runner build --delete-conflicting-outputs` and the output didn't change.